net/eds-kubelet/patches/0003-Add-shell-file-to-set-up-docker.patch

```diff
@@ -0,0 +1,72 @@
+From 09a9ef9575d8a3d947411ce860935adde2431aa2 Mon Sep 17 00:00:00 2001
+From: Yangbo Lu <yangbo.lu@nxp.com>
+Date: Fri, 8 Mar 2019 11:31:34 +0800
+Subject: [PATCH] Add shell file to set up docker
+
+Signed-off-by: Yangbo Lu <yangbo.lu@nxp.com>
+---
+ scripts/docker-setup.sh | 37 +++++++++++++++++++++++++++++++++++++
+ scripts/k8s.sh          |  3 +++
+ 2 files changed, 40 insertions(+)
+ create mode 100755 scripts/docker-setup.sh
+
+diff --git a/scripts/docker-setup.sh b/scripts/docker-setup.sh
+new file mode 100755
+index 0000000..8442f2b
+--- /dev/null
++++ b/scripts/docker-setup.sh
+@@ -0,0 +1,37 @@
++#!/bin/bash
++
++DOCKER_URL=https://download.docker.com/linux/static/stable
++DOCKER_VER=18.09.3
++DOCKER_PKG=docker.tgz
++
++which docker 1>/dev/null 2>/dev/null
++if [ $? -ne 0 ];then
++	ARCH=$(uname -m)
++	if [ ${ARCH} == "aarch64" ];then
++		ARCH=aarch64
++	elif [ ${ARCH} == "arm" ];then
++		ARCH=armhf
++	else
++		echo "${ARCH}: docker is not supported now!"
++		exit 1
++	fi
++
++	echo "Download and install docker for ${ARCH}..."
++
++	curl ${DOCKER_URL}/${ARCH}/docker-${DOCKER_VER}.tgz -o /tmp/${DOCKER_PKG}
++	tar zxvf /tmp/${DOCKER_PKG} -C /tmp 1>/dev/null 2>/dev/null
++	mv /tmp/docker/* /usr/bin
++	rm /tmp/${DOCKER_PKG} /tmp/docker -rf
```

Comment:
There is a [docker-ce](https://github.com/openwrt/packages/tree/master/utils/docker-ce) package for master. Can you use this instead of downloading Docker on-device?

Reply:
Get it. I just didn't see this package when I worked on edgescale integration.
That's great it's already there.